libkbfs/mdserver_local.go

```diff
@@ -50,6 +50,29 @@ type MDServerLocal struct {
 	shutdownLock *sync.RWMutex
 }
 
+type registerForUpdateRecord struct {
+	id       TlfID
+	currHead MetadataRevision
+}
+
+type mdServerLocalRecordingRegisterForUpdate struct {
```

Comment:
You should move this test code into a `_test.go` file, either in `kbfs_cr_test.go` or a new one you make just for this new struct.